src/main/java/org/mule/extension/aggregator/internal/privileged/executor/AbstractAggregatorExecutor.java

```diff
@@ -213,6 +222,30 @@ public void start() throws MuleException {
     }
   }
 
+  private void executeDeferActions() {
+    quorumManager = new Thread(() -> {
```

Comment:
Is this done indefinitely? Shouldn't you also add a property to configure a max number of times this is tried, otherwise fail?

Reply:
in a cluster-based solution, this must always be completed, otherwise the application may be left in an inconsistent state. As long as the quorum is not met, an attempt will be made to carry out this synchronization.

Reply:
I know but I don't think it's a good idea to keep waiting for it indefinitely. Otherwise, it there is a real problem in the cluster, you application will remain trying to be deployed forever.

Reply:
With this change, the app will deploy, but the things necessary to run as a primary node will be waiting for the quorum (if it fails for that reason). If we add a parameter with a retries limit, what happens when the app achieves this value?  The app will still be deployed but in an inconsistent state. I don't know if it is the expected behavior, I don’t think so.